modules/client-configuration/pages/supported-features.adoc

```diff
@@ -258,7 +258,7 @@ ifeval::[{uyuni-content} == true]
 | {check}
 
 | {rocky} 9
-| {x86_64}, {aarch64}, {ppc64le}, {s390x}
+| {x86_64}, {aarch64}
```

Comment:
Not right. For Rocky9 we support  x86_64, aarch64, ppc64le, s390x, at least on Uyuni :-)